pyanaconda/modules/localization/runtime.py

```diff
@@ -66,18 +66,22 @@ def run(self):
 class GetMissingKeyboardConfigurationTask(Task):
     """Task for getting missing keyboard settings by conversion and default values."""
 
-    def __init__(self, localed_wrapper, x_layouts, vc_keymap):
+    def __init__(self, localed_wrapper, live_wrapper, x_layouts, vc_keymap):
         """Create a new task.
 
         :param localed_wrapper: instance of systemd-localed service wrapper
         :type localed_wrapper: LocaledWrapper
+        :param live_wrapper: instance of LiveSystemKeyboardBase based class reflecting
```

Comment:
`live_wrapper` has again only keyboard... You mention the class for the type, but again call it something generic.

Reply:
Renamed.